src/fluree/server/consensus/network/multi_addr.clj

```diff
@@ -32,39 +32,39 @@
   (log/debug "multi-addr -> map input: " multi-addr)
   (when-not (str/starts-with? multi-addr "/")
     (invalid-address-shutdown
-      (str "Error in multi-address type in: " multi-addr)))
+     (str "Error in multi-address type in: " multi-addr)))
   (let [[_ address-type host protocol port alias? alias] (str/split multi-addr #"/")
         port* (try (Integer/parseInt port)
                    (catch Exception _
                      (invalid-address-shutdown
-                       (str "Error in multi-address port in: " multi-addr
-                            " Port value must be an integer. "
-                            "Supplied: " port))))]
+                      (str "Error in multi-address port in: " multi-addr
+                           " Port value must be an integer. "
+                           "Supplied: " port))))]
     (when-not ({"ip4" "dns4" "ip6" "dns6"} address-type)
       (invalid-address-shutdown
-        (str "Error in multi-address type in: " multi-addr
-             " Fluree only supports the ipv4, ipv6, dns4 or dns6 at the moment.")))
+       (str "Error in multi-address type in: " multi-addr
```

Comment:
i think this is an even better example of why 2 space indentation for function calls is more legible.